modules/nexx360BidAdapter.js

```diff
@@ -1,176 +1,333 @@
+    .forEach(param => video[param] = videoParams[param]);
+  return video;
+}
+
+function isVideoBid(bid) {
+  return utils.deepAccess(bid, 'mediaTypes.video');
+}
+
+function isBannerBid(bid) {
+  return utils.deepAccess(bid, 'mediaTypes.banner') || !isVideoBid(bid);
+}
+
+function toFormat(sizes) {
+  return sizes.map((s) => {
+    return { w: s[0], h: s[1] };
+  });
+}
+
+function getFloor(bid, mediaType) {
+  let floor = 0;
+  if (typeof bid.getFloor === 'function') {
+    const floorInfo = bid.getFloor({
+      currency: 'USD',
+      mediaType: mediaType,
+      size: '*'
     });
-    const payload = {
-      adUnits,
-      // TODO: does the fallback make sense here?
-      href: encodeURIComponent(bidderRequest.refererInfo.page || bidderRequest.refererInfo.topmostLocation)
-    };
-    if (bidderRequest) { // modules informations (gdpr, ccpa, schain, userId)
-      if (bidderRequest.gdprConsent) {
-        payload.gdpr = bidderRequest.gdprConsent.gdprApplies ? 1 : 0;
-        payload.gdprConsent = bidderRequest.gdprConsent.consentString;
-      } else {
-        payload.gdpr = 0;
-        payload.gdprConsent = '';
-      }
-      if (bidderRequest.uspConsent) { payload.uspConsent = bidderRequest.uspConsent; }
-      if (bidderRequest.schain) { payload.schain = bidderRequest.schain; }
-      if (userEids !== null) payload.userEids = userEids;
-    };
-    if (test) payload.test = 1;
-    const payloadString = JSON.stringify(payload);
-    return {
-      method: 'POST',
-      url: BIDDER_URL,
-      data: payloadString,
-    };
-  },
-  /**
+
+    if (typeof floorInfo === 'object' &&
+      floorInfo.currency === 'USD' &&
+      !isNaN(parseFloat(floorInfo.floor))) {
+      floor = Math.max(floor, parseFloat(floorInfo.floor));
+    }
+  }
+
+  return floor;
+}
+
+function enrichImp(imp, bid, floor) {
+  if (bid.params.customParams) {
+    utils.deepSetValue(imp, 'ext.customParams', bid.params.customParams);
+  }
+  if (floor > 0) {
+    imp.bidfloor = floor;
+    imp.bidfloorcur = 'USD';
+  } else if (bid.params.customFloor) {
+    imp.bidfloor = bid.params.customFloor;
+  }
+  if (bid.ortb2Imp && bid.ortb2Imp.ext && bid.ortb2Imp.ext.data) {
+    imp.ext.data = bid.ortb2Imp.ext.data;
+  }
+}
+
+// function createBannerRequest(bids, bidderRequest) {
+//   let data = getBaseRequest(bids[0], bidderRequest);
+//   data.imp = bids.map(bid => {
+//     const floor = getFloor(bid, BANNER);
+//     let imp = {
+//       id: bid.bidId,
+//       tagid: bid.params.unit,
+//       banner: {
+//         format: toFormat(bid.mediaTypes.banner.sizes),
+//         topframe: utils.inIframe() ? 0 : 1
+//       },
+//       ext: {divid: bid.adUnitCode}
+//     };
+
+//     if (bidderRequest.fledgeEnabled) {
+//       imp.ext.ae = bid?.ortb2Imp?.ext?.ae
+//     }
+
+//     enrichImp(imp, bid, floor);
+//     return imp;
+//   });
+//   return {
+//     method: 'POST',
+//     url: REQUEST_URL,
+//     data: data
+//   }
+// }
```

Comment:
Do we need to keep this commented function?

Reply:
Indeed, removed!